kubelet-to-gcm/Makefile

```diff
@@ -15,7 +15,7 @@
 OUT_DIR = build
 PACKAGE = github.com/GoogleCloudPlatform/k8s-stackdriver/kubelet-to-gcm
 PREFIX = staging-k8s.gcr.io
-TAG = 1.3.6
+TAG = 1.3.7
```

Comment:
IIUC this version introduces a feature, so it should bump minor, not patch version.

Reply:
OK, bumped minor version